### PR TITLE
Remove dependency on EasyBind by adding a few lines of code

### PR DESCRIPTION
**Pull request by msgilligan**
* Remove dependency on EasyBind
* Replace EasyBind with two static methods

Hopefully the code is more readable than before.

### CODE REVIEW

File: wallettemplate/src/main/java/wallettemplate/MainController.java [hunk 1]
```diff
@@ -48,6 +50,7 @@
 
     private BitcoinUIModel model = new BitcoinUIModel();
     private NotificationBarPane.Item syncItem;
+    private static MonetaryFormat monetaryFormat = MonetaryFormat.BTC.noCode();
```
Comment (schildbach) [inline wallettemplate/src/main/java/wallettemplate/MainController.java]: Perhaps make it a true constant by adding the final keyword and uppercase the name?

Comment (msgilligan) [reply wallettemplate/src/main/java/wallettemplate/MainController.java]: Will do.

Comment (msgilligan) [reply wallettemplate/src/main/java/wallettemplate/MainController.java]: Done!

**Comment by msgilligan** [comment]
Merged by @schildbach

**Pull request closed without merging**